public/gallery/index.html

```diff
@@ -87,12 +94,13 @@
 	</head>
 	<body>
     <div id="2019">
+      <div class="header_background"></div>
       <header>
         <a href="/">
-          <img src="brickhack_5.svg">
```

Comment:
Love the design change, BrickHack 6 should be BrickHack V as BH6 hasn't happened yet.